db/db_impl/db_impl_open.cc

```diff
@@ -129,6 +129,7 @@ DBOptions SanitizeOptions(const std::string& dbname, const DBOptions& src) {
   for (size_t i = 0; i < result.db_paths.size(); i++) {
     DeleteScheduler::CleanupDirectory(result.env, sfm, result.db_paths[i].path);
   }
+  DeleteScheduler::CleanupDirectory(result.env, sfm, std::string(result.wal_dir));
```

Comment:
If user leaves options.wal_dir to be empty, to we end up with adding the same trash files from DB path to the cleaning up queue twice?

Reply:
Fix that. Adding unit test is a good idea but it turns out to take more time. Will leave it for a separate PR.